### ISSUE TITLE
a3 list l

### ISSUE THREAD

**Issue by ALilalee** [ALilalee/Group-3#1]

**Comment by Selmehalawy5001** [comment]
As promised, here is a data list i have created.

https://github.com/Selmehalawy5001/ITP270-Samer/blob/main/Assignment3List1.sh

**Comment by Selmehalawy5001** [comment]
Hey ali,

What happened with this?

**Comment by ALilalee** [comment]
Can you get on discord?

**Comment by Selmehalawy5001** [comment]
Yes, I can.

**Comment by ALilalee** [comment]
@ me when you get on

**Comment by Selmehalawy5001** [comment]
10 min

**Comment by Selmehalawy5001** [comment]
I'm on. But do I need a link to join the group

**Comment by ALilalee** [comment]
in discord?

**Comment by ALilalee** [comment]
you should already be in the server

**Comment by ALilalee** [comment]
![image](https://user-images.githubusercontent.com/98132615/158906576-c05be62c-9993-45bb-8e9d-dfd3ab3fe1e0.png)

**Comment by Selmehalawy5001** [comment]
right. but for some reason im not able to join, do you have an invite link?

**Comment by ALilalee** [comment]
https://discord.gg/aPAJbHnj

**Comment by Selmehalawy5001** [comment]
Hey ali,

i tried to get on the google docs to get some work done but it says I need
access? Are you the admin?
